cGame.h

```diff
@@ -16,13 +16,13 @@
 #include <observers/cScenarioObserver.h>
 #include <data/cAllegroDataRepository.h>
 
-// forward declaration :/ sigh should really look into this
```

Comment:
haha love how you removed the comment. In hobby projects I keep my thoughts unfiltered in code haha.